src/optimization/dce.ml

```diff
@@ -50,6 +50,26 @@ let push_class dce c =
 		dce.curclass <- old
 	)
 
+let find_field c name is_static =
+	match name with
+	| "new" when not is_static ->
```

Comment:
This makes me think that instead of a bool we should have a `Static | Instance | Constructor` thing. Would be useful in other places too.